src/client/src/ui/shell/views/shell.scss

```diff
@@ -6,6 +6,11 @@ $workspace-height:            calc(100vh - #{$header-height} - #{$blotter-height
 $analytics-width:             400px;
 $workspace-width:             calc(100vw - #{$analytics-width});
 
+@font-face {
+  font-family: 'Rubik Light';
+  src: url('../../../resources/fonts/RubikLight.woff2');
+}
+
```

Comment:
i have created a _fonts.scss file in my other PR. Might be good moving all the fonts registration there once its merged